src/main/java/io/github/apickledwalrus/skriptgui/SkriptGUI.java

```diff
@@ -22,12 +22,22 @@ public class SkriptGUI extends JavaPlugin {
 	@Override
 	public void onEnable() {
 		Plugin skript = getServer().getPluginManager().getPlugin("Skript");
-		if (skript == null || !skript.isEnabled()) {
-			getLogger().severe("Could not find Skript! Make sure you have it installed and that it properly loaded. Disabling...");
+		Version minimumSupportedVersion = new Version(2, 6, 3);
+		if (skript == null) {
+			// Skript doesn't exist within the server plugins folder
+			getLogger().severe("Could not find Skript! Make sure you have it installed. Disabling...");
+			getLogger().severe("skript-gui requires Skript 2.6.3 or newer! Download Skript releases at https://github.com/SkriptLang/Skript/releases");
```

Comment:
```suggestion
			getLogger().severe("skript-gui requires Skript " + minimumSupportedVersion + " or newer! Download Skript releases at https://github.com/SkriptLang/Skript/releases");
```
Meant to suggest this before - oops